.github/workflows/publish.yaml

```diff
@@ -20,11 +16,16 @@ concurrency:
 jobs:
   publish-python:
     runs-on: ubuntu-22.04  # convco needs GLIBC_2.32 which is not in 20.04
-    if: false
     environment: publish
     outputs:
       new_version: ${{ steps.set-vars.outputs.new_version }}
     steps:
+      - name: Check ref
```

Comment:
This part is not needed, since the `publish` environment, which is used for this job, is set up to only run on the `main` branch.